docs/contributing/Developing a Language Feature.md

```diff
@@ -0,0 +1,67 @@
+  
+    *Note*: The goal is not to have a 100%-complete design but instead a general understanding of how the language feature is intended to work.
+
+1. **Discussion**: The goal of the discussion is to help clarify the design and determine if the Language Team wants to sanction the proposal. As the discussion evolves the author is encouraged to update the speclet to incorporate the changing design.  This allows folks who join later to understand the most recent proposal by reading the top of the thread.  
+
+1. **Language team makes a decision**: When the Language Team believes enough information is available they will update the issue accordingly:
+
+    * **Ready for prototype**: The feature is worth pursuing as a prototype for the language. 
+    * **Not actionable**: The discussion didn't lead to a design that was actionable by the team. 
+    * **Declined as proposed**: The proposal is overall a good idea but the proposed implementation details are not as efficient as they could be. In most cases, the language team will attempt to steer the discussion to a cleaner design but understand that it may be simpler to close and start over with a new issue.  
+    * **Backlog**: The feature is a good idea with an acceptable design but the Language Team simply can't allocate the resources at this time for this work. 
+    * **Won't Fix**: The feature isn't taking the language in a direction the Language Team is interested in pursuing.  
+    
+    A Language Team member will be paired with a community-driven feature when it is deemed 'Ready for prototype'. This team member will act as the primary source of contact for the community and will help drive the feature to completion.  
+    
+1. **Prototype phase**: All features are initially prototyped in a separate branch. The name of the branch is `features/<feature name>`. These branches are created when a feature speclet is marked as 'Ready for prototype', language team member champions the feature, and [a developer is assigned](#community-contributions) to implement the feature. The developer is responsible for keeping an up-to-date speclet of the feature at `docs/features/<feature name>.md` throughout the development.  
+ 
+     All features should be off by default until fully accepted into the language. A new feature should only be enabled when explicitly included in the `/features:<feature-name>` compiler option. This helps ensure the language is always in a shippable state no matter what prototypes are in the current branch. 
+
+1. **Prototype decision**: When the prototype phase ends, the Language Team will make a decision on the feature: 
+    * **Ready to integrate**: The feature implementation revealed no blocking issues, has a healthy set of test suites, and is sufficiently complete in the compiler, IDE, debugger, etc. to integrate into the main tree on a selected branch.  
```

Comment:
Should you add something about test suites showing the impact (or proving lack of impact) on other language features?